### PR TITLE
Article categories for external news

### PR DESCRIPTION
**Pull request by AlbertStjarne**
## PT:
https://www.pivotaltracker.com/story/show/161230458

## Included in PR
* Adding buttons for each category in the news-api
* Clicking button fetches correct news
* Styling

## What we learnt
* Using JavaScript to listen to button

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,3 +14,13 @@
...
+
+function openDiv(categories) {
+    let category = document.querySelectorAll(".category");
+        category.forEach( id => id.style.display = "none");
+        document.getElementById(categories).style.display = "block";
+}
+
+document.addEventListener('turbolinks:load', ()=>{
+    App.windowContent = document.getElementById('content')
+})
```
Comment (Rinbo) [inline app/assets/javascripts/application.js:20]: If I am reading the code correctly it seems like where are not using this function anymore?

File: app/views/partials/_categoriesAPI.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <option value="health">Health</option>
+    <option value="science">Science</option>
+    <option value="sports">Sports</option>
+    <option value="technology">Technology</option>
+</select>
+
+<script>
+
+const CAT = () =>  {
+    let category = document.getElementById("myChoice").value
```
Comment (Rinbo) [inline app/views/partials/_categoriesAPI.html.erb]: Where is "myChoice" taken from?

**Review by Noelryn — COMMENTED** [review]
Good job, Zachary!! Looks good to me

File: app/views/main/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,7 @@
 <section class="grid-container">
-  <div class="top-news"> <%= render 'partials/toparticle' %></div>
-  <div class="news-feed py-2 px-4"><%= render 'partials/apifeed' %></div>
+  <div class="breaking-news"> <%= render 'partials/toparticle' %></div>
```
Comment (Noelryn) [inline app/views/main/index.html.erb:2]: I had to change breaking news to top news to pass tests are the tests still passing

**Review by Noelryn — APPROVED** [review]

**Review by Rinbo — APPROVED** [review]

**Pull request merged**